io/jvm/src/main/scala/fs2/io/ioplatform.scala

```diff
@@ -137,4 +138,29 @@ private[fs2] trait ioplatform {
   def stdinUtf8[F[_]: Sync](bufSize: Int): Stream[F, String] =
     stdin(bufSize).through(text.utf8.decode)
 
+  /** Stream of bytes read asynchronously from the specified resource relative to the class `C`.
+    * @see [[readClassLoaderResource]] for a resource relative to a classloader.
+    */
+  def readClassResource[F[_], C](
+      name: String,
+      chunkSize: Int
+  )(implicit F: Sync[F], ct: ClassTag[C]): Stream[F, Byte] =
+    Stream.eval(F.blocking(Option(ct.runtimeClass.getResourceAsStream(name)))).flatMap {
+      case Some(resource) => io.readInputStream(resource.pure, chunkSize)
+      case None           => Stream.raiseError(new IOException(s"Resource $name not found"))
+    }
+
+  /** Stream of bytes read asynchronously from the specified classloader resource.
+    * @see [[readClassResource]] for a resource relative to a class.
+    */
+  def readClassLoaderResource[F[_]](
+      name: String,
+      chunkSize: Int,
```

Comment:
Default chunk size here and above to 64K like in Files?

Reply:
```suggestion
      chunkSize: Int = 64 * 1024,
```